### ISSUE TITLE
phoemplocal - Test and document

### ISSUE THREAD

**Issue by ascbot** [DOI-USGS/ISIS3#989]
---

Author Name: **Stuart Sides** (@scsides)

Original Assignee: Ella (Contractor) Lee

---

Test and document phoemplocal for release

*Additional information:*

Changed Target Version out to August for now (a more realistic target considering all that's on the Isis and developer plate right now)

**Comment by ascbot** [comment]
---

Original Redmine Comment
Author Name: **Tammy Becker** (Tammy Becker)
Original Date: 2012-09-19T23:44:18Z

---

Once documentation for photomet and photemplate are complete, this will help test and document this app.  Some additional documentation will be needed for the specific functionality.

Unassigned from janet for now; based on new funding and direction.

**Comment by ascbot** [comment]
---

Original Redmine Comment
Author Name: **Tammy Becker** (Tammy Becker)
Original Date: 2012-11-13T20:42:03Z

---

Ella will update the documentation to be consistent with the most recent photomet updates under the Q1-ISIS support.

**Comment by ascbot** [comment]
---

Original Redmine Comment
Author Name: **Ella (Contractor) Lee** (Ella (Contractor) Lee)
Original Date: 2013-01-29T01:06:22Z

---

<B>IMPACT</B>
The updates to the documentation should not impact other programs.

**Comment by ascbot** [comment]
---

Original Redmine Comment
Author Name: **Ella (Contractor) Lee** (Ella (Contractor) Lee)
Original Date: 2013-01-29T01:19:38Z

---

This section about the need for a new software was removed and added to mantis #875, because it needed to be documented in mantis instead of the program documentation.

This program should serve as a clear example for developing a new program 
to fit the parameter of an empirical photometric function to part of an
actual image. The differences can be summarized as follows:

1) Fit is done only once (not in a table vs. phase angle)
   since the input image has a fixed geometry.
2) The data to be fit *to* are obtained from an ISIS cube plane
   rather than modeled in in the code.
3) Similarly, the incidence, emission, and phase angles are
   obtained from backplanes of the same cube and maintained
   in buffers corresponding point-for-point with the image.
4) Both image data and angles should be converted to double
   precision before being used.
5) Optionally, a region-of-interest mask could be obtained
   from a backplane and only those pixels in the ROI used
   in the fit.  This would be in addition to the incidence
   and emission angle tests already done in the code.

Let me know if there is a strong desire to keep it in the documentation.

Ella Mae

**Comment by ascbot** [comment]
---

Original Redmine Comment
Author Name: **Ella (Contractor) Lee** (Ella (Contractor) Lee)
Original Date: 2013-01-29T01:21:05Z

---

The documentation for phoemplocal has been updated.  If you would like to review the documentation use the following:

firefox /work/projects/isis/latest/m00450/isis/src/base/apps/phoemplocal/phoemplocal.html
